content/enterprise/admin/application/general.mdx

```diff
@@ -34,7 +34,7 @@ When new user accounts are created, if they cannot create their own organization
 
 ## API Rate Limiting
 
-By default, requests to the Terraform Cloud API from a single user or IP address are [limited to 30 requests per second](/cloud-docs/api-docs#rate-limiting) to prevent abuse or hogging of resources. Since usage patterns may vary for a given instance, this can be updated to match local needs.
+By default, requests to the Terraform Cloud API from a single user or IP address are [limited to 30 requests per second](/cloud-docs/api-docs#rate-limiting) to prevent abuse or hogging of resources. Since usage patterns may vary for a given instance, this can be updated to match local needs. A few endpoints have lower limits to prevent email spam attack vectors. If you receive a rate limited response, the limit will be reflected in the `x-ratelimit-limit` header once triggered.
```

Comment:
```suggestion
By default, requests to the Terraform Cloud API from a single user or IP address are [limited to 30 requests per second](/cloud-docs/api-docs#rate-limiting) to prevent abuse or hogging of resources. Since usage patterns may vary for a given instance, this can be updated to match local needs. A few endpoints have lower limits to prevent certain spam and abuse scenarios. If you receive a rate limited response, the limit will be reflected in the `x-ratelimit-limit` header once triggered.
```